MINIKUBE.md

```diff
@@ -0,0 +1,170 @@
+# local development with minikube
+
+1. clone and run [razeedash](https://github.com/razee-io/Razeedash.git)
+
+```bash
+    git clone https://github.com/razee-io/Razeedash.git
+    cd razeedash
+    meteor npm install
+    meteor
+```
+
+1. clone and run [razeedash-api](https://github.com/razee-io/Razeedash-api.git)
+
+```bash
+    git clone https://github.com/razee-io/Razeedash-api.git
+    cd razeedash-api
+    npm install
+    npm start
+```
+
+1. edit your hosts file (/etc/hosts on mac and linux) and add:
+
+```bash
+    192.168.99.1 minikube-host
+```
+
+1. [Install minikube on your laptop](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+
+For macs:
+
+```bash
+    brew cask install virtualbox
+    brew cask install minikube
+    minikube start
+    😄  minikube v1.1.0 on darwin (amd64)
+    💡  Tip: Use 'minikube start -p <name>' to create a new cluster, or 'minikube delete' to delete this one.
+    🔄  Restarting existing virtualbox VM for "minikube" ...
+    ⌛  Waiting for SSH access ...
+    🐳  Configuring environment for Kubernetes v1.14.2 on Docker 18.09.6
+    🔄  Relaunching Kubernetes v1.14.2 using kubeadm ...
+    ⌛  Verifying: apiserver proxy etcd scheduler controller dns
+    🏄  Done! kubectl is now configured to use "minikube"
+```
+
+1. Configure minikube
+
+- Make sure your KUBECONFIG is pointing to your minikube cluster
+- `kubectl create namespace razee`
+- kubectl apply this yaml:
+
+```yaml
+    kind: Service
```

Comment:
Snippet has an extra space after 1st line when cutting and pasting to file that makes the indents messed up.  Can that be fixed?